lib/services/user/user_service.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:survey/models/user_info.dart';
+import 'package:survey/services/api/api_service.dart';
+import 'package:survey/services/http/http_service.dart';
+import 'package:survey/services/locator/locator_service.dart';
+
+abstract class UserService {
+  Future<UserInfo> me();
+}
+
+class UserServiceImpl implements UserService {
+  final ApiService _apiService = locator.get();
+
+  @override
+  Future<UserInfo> me() {
```

Comment:
maybe getProfile to be clearer

Reply:
@sleepylee 
The word `me` here is fine IMO.
Also, the guy who defines the API path which uses `/me` too 😁

Reply:
@markgravity yes, but a function should start with `verb`, at least adding `getMe()` or sth like that. How do you think?

Reply:
@manh-t ok, I end up with `getProfile` would be better than `getMe` . Googbye my `me` method 😢